src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkUniIslService.java

```diff
@@ -66,6 +68,7 @@ public void uniIslDiscovery(Endpoint endpoint, IslInfoData speakerDiscoveryEvent
         IslDataHolder islData = new IslDataHolder(speakerDiscoveryEvent);
         if (reference.equals(effectiveReference)) {
             carrier.notifyIslUp(endpoint, reference, islData);
+            sendSlowDiscoveryDisableRequest(endpoint);
```

Comment:
You don't need it here... here is a "normal" not fist discovery poll notification proxied into ISL. You need to produce such mode switching only on "isl" changes... i.e. somewhere below line 87 `endpointData.put(endpoint, effectiveReference);`.

Reply:
Removed